static/css/base.scss

```diff
@@ -21,6 +21,9 @@ $full-footer-bg: #fafcff;
 
 :root {
   --font-family: "Roboto", "Helvetica Neue", Arial, sans-serif !important;
+  --font-family-sans-serif: "Google Sans Text", -apple-system, BlinkMacSystemFont,
```

Comment:
Maybe use the variable `$font-family-sans-serif` here?

Reply:
actually, don't need to redefine it here because setting $font-family-sans-serif already defines it as a variable!